Obsidian/Daratrine/03_Narrative/03.1_Macro_History/História de Daratrine (Inteira).md

```
@@ -0,0 +1,160 @@
+- os anões perderam o controle da situação.
+
+---
+
+## Arcaror: O Reino Élfico
+
+Arcaror era o principal reino dos elfos.
+Características:
+- exímios manipuladores de magia;
+- forte ligação entre natureza, magia e religião;
+- sociedade guiada pela espiritualidade.
+##### A Profecia de Elmartin
```

markdownlint-cli2 (0.22.1)

[warning] Heading levels should only increment by one level at a time
Expected: h3; Actual: h5

(MD001, heading-increment)